libs/portals/shared-modules/delegations/src/hooks/useDomains/useDomains.ts

```diff
@@ -80,17 +75,15 @@ export const useDomains = (includeDefaultOption = true) => {
     const option = opt ?? allDomainsOption
     const name = option.value
     setDomainName(name)
-    sessionStore.setItem('domain', name)
 
-    const query = new URLSearchParams(location.search)
-    const currentDomain = query.get('domain')
+    const query = new URLSearchParams({ domain: name })
 
-    if (currentDomain && currentDomain !== displayNameQueryParam) {
-      query.set('domain', name)
-      navigate(`${location.pathname}?${query.toString()}`, {
-        replace: true,
-      })
-    }
+    const domainQuery = name !== ALL_DOMAINS ? `?${query.toString()}` : ``
```

Comment:
Normal single quotes when there is no string interpolation.
```suggestion
    const domainQuery = name !== ALL_DOMAINS ? `?${query.toString()}` : ''
```